great-idea-website/index.html

```diff
@@ -16,48 +16,67 @@
 
   <body>
 
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
+    <header>
 
-  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
+      <nav>
+          <a href="#">Services</a>
```

Comment:
Nice nav bar!

Reply:
> Im not sure how you tried to pull this, did you make your own branch in the terminal and if so maybe you just tried to merge from the master and not your own branch because I do not see a pull request out of your own branch. If you need help with this part later just give me a shout.
> 

I am honestly not that sure either. I just sort of fumbled through the instructions I got from the youtube vid in the TK and the instructions that were dropped in slack. Thursday when we learn Git I will make sure to pay very close attention. Please let me know if I do weird stuff like that again after Thursday.